examples/stable_baselines3/sb3_example.ipynb

```diff
@@ -0,0 +1,274 @@
+        "            \"Continuous\",\n",
+        "        ),\n",
+        "        done_criteria=smarts_agent_interface.DoneCriteria(\n",
+        "            collision=True,\n",
+        "            off_road=True,\n",
+        "            off_route=False,\n",
+        "            on_shoulder=False,\n",
+        "            wrong_way=False,\n",
+        "            not_moving=False,\n",
+        "        ),\n",
+        "    )\n",
+        "\n",
+        "    agent_specs = {\n",
+        "        \"Agent-007\": smarts_agent.AgentSpec(\n",
+        "            interface=vehicle_interface,\n",
+        "            agent_builder=None,\n",
+        "        )\n",
+        "    }\n",
+        "\n",
+        "    env = smarts_hiway_env.HiWayEnv(\n",
+        "        scenarios=scenarios,\n",
+        "        agent_specs=agent_specs,\n",
+        "        headless=headless,\n",
+        "        visdom=False,\n",
+        "        seed=seed,\n",
+        "        sim_name=sim_name,\n",
+        "    )\n",
+        "\n",
+        "    # Wrap env with ActionWrapper\n",
+        "    env = action.Action(env=env)\n",
+        "    # Wrap env with RewardWrapper\n",
+        "    env = reward.Reward(env=env)\n",
+        "    # Wrap env with RGBImage wrapper to only get rgb images in observation\n",
+        "    env = smarts_rgb_image.RGBImage(env=env, num_stack=1)\n",
+        "    # Wrap env with SingleAgent wrapper to be Gym compliant\n",
+        "    env = smarts_single_agent.SingleAgent(env=env)\n",
+        "    env = monitor.Monitor(env=env)\n",
+        "    check_env(env, warn=True)\n",
+        "\n",
+        "    # create the model\n",
+        "    model = PPO(\"CnnPolicy\", env, verbose=1)\n",
+        "\n",
+        "    # evaluate at the beginning\n",
+        "    before_mean_reward, before_std_reward = evaluate_policy(\n",
+        "        model, env, n_eval_episodes=10, deterministic=True\n",
+        "    )\n",
+        "    model.learn(total_timesteps=500000)\n",
+        "\n",
+        "    # evaluate after training\n",
+        "    mean_reward, std_reward = evaluate_policy(\n",
+        "        model, env, n_eval_episodes=10, deterministic=True\n",
+        "    )\n",
+        "\n",
+        "    print(\n",
+        "        f\"before_mean_reward:{before_mean_reward:.2f} +/- {before_std_reward:.2f}\"\n",
+        "    )\n",
+        "    print(f\"mean_reward:{mean_reward:.2f} +/- {std_reward:.2f}\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "s3JVKDRdyANx"
+      },
+      "source": [
+        "Run the example"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "-0MXABlGxiIC",
+        "outputId": "85f9a621-8777-4ad0-8bf2-e6898146c559"
+      },
+      "outputs": [],
+      "source": [
+        "# allow offscreen render\n",
```

Comment:
I believe Colab loses its attractiveness without visualization. 

Since we are unable to get Envision to work in Colab now, I would suggest removing the Colab `.ipynb` example from this pull request.

We may add the Colab `.ipynb` example file later, when we manage to get the Envision visualization to work in Colab.

Reply:
I disagree here, detailed visualisation is useful but the core issue is to make SMARTS accessible. Some of the examples I have seen do not have visualisation past console output. The other half of what the notebook does well is to help interleave documentation with example.

Reply:
I have temporarily removed it for now, but I can add it again in the future easily